default/stringtables/en.txt

```diff
@@ -5032,7 +5032,14 @@ SITREP_SNOOZE_CLEAR_INDEFINITE
 '''  Show all SitReps indefinitely dismissed'''
 
 SITREP_SNOOZE_CLEAR_ALL
-'''  Show all hidden or dismissed SitReps'''
+'''  Show all dismissed SitReps'''
+
+# %1% localized name of SitRep template
+SITREP_HIDE_TEMPLATE
+'''  Hide all SitReps with template: %1%'''
```

Comment:
Changed to `SitReps of type`, to be in-line with #1019.
I was unaware of the use of `label`, will search for it.